android-json-form-wizard/src/main/java/com/vijay/jsonwizard/utils/FormUtils.java

```diff
@@ -2100,6 +2100,14 @@ public BufferedReader getRulesFromRepository(@NonNull Context context, @NonNull
         }
 
         ClientFormContract.Model clientForm = clientFormDao.getActiveClientFormByIdentifier(localeFormIdentity);
+        if (clientForm == null) {
+            // Strip anything before the '/'
+            if (StringUtils.isNotBlank(fileName) && fileName.contains("/")) {
+                localeFormIdentity =  localeFormIdentity.split("/")[1];
```

Comment:
this might throw and error incase we do not have index 1. It's highly unlikely though

Reply:
@dubdabasoduba Added a check for this